vision/vision.proto

```diff
@@ -161,6 +161,37 @@ message RectangularDetection {
   bytes image_small = 7;
 }
 
+// Collection of images used as source for
+// a detection
+message ImageList {
+  // image data 
+  repeated bytes data = 1;
+
+  // if image data above was taken in a sequence, the fps should 
+  // describe how many of such frames were taken in a second
+  int32 fps = 2;
```

Comment:
I would use frames_per_second here. It's better not to use abbreviations. This way you can spare the comments in code.